### PR TITLE
Keyword Add and updates

### PR DESCRIPTION
**Pull request by nesrineBa-craft**
Added new method for checking attributes in dic : Check_Response_Attribute_Dictionnaries
updated check all attributes method : Check_All_Response_Attribute

### CODE REVIEW

File: Referential/Minimal/1_Tags/Tags.robot [hunk 1]
```diff
@@ -1,5 +1,5 @@
 *** Settings ***
-Resource              ${EXECDIR}/Requests/Tags.robot
+Resource              ../../../Requests/Tags.robot
```
Comment (zied-cf) [inline Referential/Minimal/1_Tags/Tags.robot:2]: @nesrineBa-craft does it work ??

I think it's  ../.. (only 2 levels)

File: Referential/Minimal/2_Relation/Relations.robot [hunk 2]
```diff
@@ -1,5 +1,5 @@
 *** Settings ***
-Resource                  ${EXECDIR}/Requests/Relation.robot
+Resource                  ../../../Requests/Relation.robot
```
Comment (zied-cf) [inline Referential/Minimal/2_Relation/Relations.robot:2]: same

File: Referential/Minimal/4_Sources/Sources.robot [hunk 3]
```diff
@@ -1,5 +1,5 @@
 *** Settings ***
-Resource              ${EXECDIR}/Requests/Sources.robot
+Resource              ../../../Requests/Sources.robot
```
Comment (zied-cf) [inline Referential/Minimal/4_Sources/Sources.robot:2]: same

File: Referential/Minimal/5_Users/Data.py [hunk 4]
```diff
@@ -502,6 +503,33 @@
...
+    "lastName": "Ba",
+    "gender": "F",
+    "image": {
+      "type": "EXTERNAL_URL",
+      "reference": "http://lorempixel.com/40/40"
+    }
+  },
+  "tags": [
+      "7C7HHTM5"
+  ],
```
Comment (zied-cf) [inline Referential/Minimal/5_Users/Data.py:523]: Tags are not supposed to be static, please check

Comment (zied-cf) [reply Referential/Minimal/5_Users/Data.py:523]: and coordinate with Anis

File: Referential/Minimal/8_KPIs/data.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+Headers = {
+    'Content-Type': 'application/json'
+    }
+
+Endpoint = 'apis/kpiengine-api/v2/kpiDefinitions'
```
Comment (zied-cf) [inline Referential/Minimal/8_KPIs/data.py:5]: please leave an empty line at end of file

**Review by zied-cf — COMMENTED** [review]
@nesrineBa-craft Please check my comments

File: Referential/Minimal/1_Tags/Tags.robot (hunk 1 above)
Comment (nesrineBa-craft) [reply Referential/Minimal/1_Tags/Tags.robot:2]: It does work i tested it already.

File: Referential/Minimal/5_Users/Data.py (hunk 4 above)
Comment (nesrineBa-craft) [reply Referential/Minimal/5_Users/Data.py:523]: yes i checked with him he will update it all the data in Users / Departments @anisbmamou

File: Referential/Minimal/1_Tags/Tags.robot (hunk 1 above)
Comment (zied-cf) [reply Referential/Minimal/1_Tags/Tags.robot:2]: Forget what I said, I just checked

**Pull request merged**